src/platform/plugins/shared/unified_doc_viewer/public/components/observability/traces/doc_viewer_overview/overview.tsx

```diff
@@ -112,7 +114,11 @@ export function Overview({
               docId={docId}
             />
           ) : null}
+          <EuiSpacer size="m" />
           {docId ? <SpanLinks traceId={traceId} docId={docId} /> : null}
+          <EuiSpacer size="m" />
```

Comment:
No for now, but instead of adding many `<EuiSpacer size="m" />` we should use <EuiFlexGroup>